exporter/elasticsearchexporter/factory.go

```diff
@@ -206,16 +202,32 @@ func exporterhelperOptions(
 		exporterhelper.WithCapabilities(consumer.Capabilities{MutatesData: false}),
 		exporterhelper.WithStart(start),
 		exporterhelper.WithShutdown(shutdown),
-		exporterhelper.WithQueue(cfg.QueueSettings),
 	}
+	qs := cfg.QueueSettings
 	if cfg.Batcher.enabledSet {
-		opts = append(opts, exporterhelper.WithBatcher(cfg.Batcher.BatcherConfig)) //nolint:staticcheck
+		if cfg.Batcher.Enabled {
```

Comment:
```suggestion
		qs.Sizer = exporterhelper.RequestSizerTypeItems // TODO: Delete once core dependency updated.
		if cfg.Batcher.Enabled {
```
This is the fix to integration tests. See explanation below